### PR TITLE
Local testing with docker compose and use it by default

### PR DESCRIPTION
**Pull request by sakama**
Same with https://github.com/embulk/embulk-input-ftp/pull/18

Also disables FTPS related test cases since setup both of FTP/FTPS services in same docker image is difficult.

### CODE REVIEW

File: src/test/java/org/embulk/output/ftp/TestFtpFileOutputPlugin.java [hunk 1]
```diff
@@ -162,76 +162,77 @@ public void testTransactionWithInvalidHost()
...
-                .set("path_prefix", "my-prefix")
-                .set("file_ext", ".csv")
-                .set("formatter", formatterConfig());
-
-        Schema schema = config.getNested("parser").loadConfig(CsvParserPlugin.PluginTask.class).getSchemaConfig().toSchema();
-
-        runner.transaction(config, schema, 0, new Control());
-    }
+    // Disable FTPS related test
+//    @Test
```
Comment (kamatama41) [inline src/test/java/org/embulk/output/ftp/TestFtpFileOutputPlugin.java:166]: `@Ignore` should be used rather than commenting out.

**Comment by kamatama41** [comment]
Basically LGTM 👍

**Comment by kamatama41** [comment]
I think FTPS related tests can work by running another FTPS container but currently `mcreations/ftp` image doesn't seem to support FTPS. I will try to create a patch when I have a spare time.

**Review by kamatama41 — APPROVED** [review]

**Comment by sakama** [comment]
Thanks!

**Pull request merged**